scripts/lib.star

```diff
@@ -861,15 +860,16 @@ def deploy_to_kubernetes_step(edition, is_downstream=False):
         ],
     }
 
+def edition_sfx(edition):      
+    if edition == 'enterprise2':
+        return '-{}'.format(edition)
+    return ''
+
 def upload_packages_step(edition, ver_mode, is_downstream=False):
     if ver_mode == 'master' and edition in ('enterprise', 'enterprise2') and not is_downstream:
         return None
-
-    sfx = ''
-    packages_bucket = ''
-    if edition == 'enterprise2':
-        sfx = '-{}'.format(edition)
-        packages_bucket = ' --packages-bucket grafana-downloads' + sfx
+    
+    packages_bucket = ' --packages-bucket grafana-downloads' + edition_sfx(edition)
```

Comment:
I'm not sure this change is necessarily a good idea, because it changes functionality. I.e., currently for other editions than enterprise2, the default bucket is used. With this change, it's assumed that the correct bucket for other editions than enterprise2 is "grafana-downloads". That assumption _could_ break at some point.

Being specific about the bucket _could_ be a good change, but it's not just a code refactoring.

Reply:
I think if it did break it we can update it here, better to be explicit than to have hidden defaults that are sometimes used and sometimes overriden?

Reply:
I'm actually unsure if it's a good idea or not.